spec/features/workflow/unschedule_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.feature "Unschedule an edition" do
+  scenario do
+    given_there_is_a_scheduled_document
+    when_i_visit_the_summary_page
+    and_i_click_stop_scheduled_publishing
+    then_the_document_is_no_longer_scheduled_and_the_status_reverted
+    and_the_scheduled_datetime_is_still_persisted
+  end
+
+  def given_there_is_a_scheduled_document
+    schedule = create(:scheduling)
+    @datetime = Time.current.tomorrow.at_noon
+    @edition = create(:edition,
+                      :scheduled,
+                      scheduling: schedule,
+                      scheduled_publishing_datetime: @datetime)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_i_click_stop_scheduled_publishing
+    click_on "Stop scheduled publishing"
+  end
+
+  def then_the_document_is_no_longer_scheduled_and_the_status_reverted
+    expect(@edition.status.details.pre_scheduled_status.draft?).to eq(true)
+    expect(@edition.scheduled?).to eq(true)
+
+    @edition.reload
+
+    expect(@edition.draft?).to eq(true)
```

Comment:
Given these are feature tests, we should be ok to see from the UI that the edition has reverted to its pre-scheduled status.  Something like `expect(page).to have_content(I18n.t!("user_facing_states.submitted_for_review.name"))` and that the timeline entry appears (see https://github.com/alphagov/content-publisher/blob/082d1bcf33523b8dce9ed458f353f175bb6bce16/spec/features/workflow/clear_scheduled_publishing_datetime_spec.rb#L59 for an example) should be enough here.

Reply:
Sure, i originally only did db level checks as i didn't know how drastically the design would change and so how much the frontend would change, but now we know i can make those changes :)